specs/~inheritance.yml

```diff
@@ -0,0 +1,225 @@
+overview: |
+  Like partials, Parent tags are used to expand an external template into the
+  current template. Contrary to partials, Parent tags may contain optional
+  arguments delimited by block tags. For this reason, Parent tags may also be
+  referred to as Parametric Partials.
+
+  The Parent tags' content MUST be a non-whitespace character sequence NOT
+  containing the current closing delimiter; each Parent tag MUST be followed by
+  an End Section tag with the same content within the matching Parent tag.
+
+  This tag's content names the Parent template to inject. Set Delimiter tags
+  MUST NOT affect the parsing of a Parent template. The Parent MUST be rendered
+  against the context stack local to the tag. If the named Parent cannot be
+  found, the empty string SHOULD be used instead, as in interpolations.
+
+  Parent tags SHOULD be treated as standalone when appropriate. If this tag is
+  used standalone, any whitespace preceding the tag should be treated as
+  indentation, and prepended to each line of the Parent before rendering.
+
+  The Block tags' content MUST be a non-whitespace character sequence NOT
+  containing the current closing delimiter. Each Block tag MUST be followed by
+  an End Section tag with the same content within the matching Block tag. This
+  tag's content determines the parameter or argument name.
```

Comment:
Because of the collision with "argument" and "parameter" in programming languages, it feels like picking an entirely different thing to call these would be preferable. Can we use "block name" rather than "parameter or argument" to avoid the ambiguity around first/last override winning? 

If so, should we not call them parametric partials / parameterized partials / etc at all?

Reply:
What do you mean by "collision"? Parents with blocks behave pretty much exactly like functions with (default) arguments. See also my discussion with @gasche at the bottom of #75.